packages/owl-bot/src/configs-store.ts

```diff
@@ -87,4 +87,14 @@ export interface ConfigsStore {
     lock: OwlBotLock,
     pullRequestId: string
   ): Promise<string>;
+
+  /**
+   * Finds repositories who list one of the changed files as a source in
+   * copy-files.
+   * @param changedFilePaths file paths in googleapis that changed.
+   *   ex: ["/google/cloud/vision/v1/vision-v1-nodejs/src/v1/image_annotator_client.ts"]
+   * @returns the list of repo names.
+   *   ex: ["googleapis/nodejs-vision", "googleapis/python-vision"]
+   */
+  findReposAffectedByFileChanges(changedFilePaths: string[]): Promise<string[]>;
```

Comment:
Am I understanding the algorithm correctly that when we get an event triggered by an update to `googleapis-gen`, we need to:

1. get a list of all paths affected by that push.
2. run this method to get repos affected.
3. dispatch a PubSub job for each repo, which contains the `repo`, and `sha` of change?

Reply:
Exactly.